src/plot_api/plot_api.js

```diff
@@ -205,7 +205,7 @@ Plotly.plot = function(gd, data, layout, config) {
             if(trace.visible !== true || !trace._module.colorbar) {
                 Plots.autoMargin(gd, 'cb' + trace.uid);
             }
-            else trace._module.colorbar(gd, cd);
+            else if(trace.marker.colorbar) trace._module.colorbar(gd, cd);
```

Comment:
I did not find any problems and tried again today, the local test passed this time. Not sure why it failed last time. So I pushed the commit you suggested.